lib/state/electron/middleware.ts

```diff
@@ -90,15 +90,27 @@ export const middleware: S.Middleware = ({ dispatch, getState }) => {
     }
   });
 
-  window.electron.send('settingsUpdate', getState().settings);
+  window.electron.send('appStateUpdate', {
+    settings: getState().settings,
+    editMode: getState().ui.editMode,
+  });
 
   return (next) => (action) => {
     const prevState = getState();
     const result = next(action);
     const nextState = getState();
 
     if (prevState.settings !== nextState.settings) {
-      window.electron.send('settingsUpdate', nextState.settings);
+      window.electron.send('appStateUpdate', {
+        settings: nextState.settings,
+        editMode: nextState.ui.editMode,
+      });
+    }
+    if (prevState.ui.editMode !== nextState.ui.editMode) {
+      window.electron.send('appStateUpdate', {
+        settings: nextState.settings,
+        editMode: nextState.ui.editMode,
+      });
```

Comment:
there's no need to separate these `if` blocks. we could instead combine the conditions

```ts
if (
	prevState.settings !== nextState.settings ||
	prevState.editMode !== nextState.editMode
) {
	window.electron.send('appStateUpdate', { … } );
}
```